tfjs-backend-webgpu/src/setup_test.ts

```diff
@@ -27,65 +27,30 @@ import {parseTestEnvFromKarmaFlags, setTestEnvs, setupTestFilters, TEST_ENVS, Te
 
 const TEST_FILTERS: TestFilter[] = [
   {
-    include: 'less',
-  },
-  {
-    include: 'clip',
-    excludes: [
-      'gradient',  // gradient function not found.
-    ]
-  },
-  {
-    include: 'greater',
-    excludes: [
-      'gradient',  // gradient function not found.
-    ]
-  },
-  {
-    include: 'div',
-    excludes: [
-      'gradient',  // gradient function not found.
-    ]
-  },
-  {
-    include: 'depthwise',
-    excludes: [
-      'gradient',   // depthwiseConv2DDerInput not yet implemented.
-      'leakyrelu',  // Not yet implemented.
-    ]
-  },
-  {
-    include: 'fused conv2d',
+    startsWith: 'abs ',
     excludes: [
-      'gradient x=[2,3,3,1] f=[2,2,1,1] s=1 p=0',  // conv2dDerInput not yet
-                                                   // implemented
-      'backProp',   // Conv2DBackpropFilter not yet
-                    // implemented
-      'leakyrelu',  // Not yet implemented
+      'complex64',  // Kernel 'ComplexAbs' not registered.
+      '5D',         // Rank 5 is not yet implemented.
```

Comment:
Can we remove all the '5D' and '6D' here given they're in excludes list?

Reply:
Should not in current filter logic (https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/jasmine_util.ts#L141-L160)
When a filter returns true, other filters behind it will not be executed.
That means the general EXCLUDE filter has no chance to filter `5D` cases, STARTSWITH filters have to handle this itself.

Reply:
We may add below in the top of list so that we can handle all the 5D and 6D cases in the same place.
{
include: '5D',
excludes: [' webgpu ']
}

Reply:
Does your point is to make this as the first filter
{
include: '5D',
excludes: [' webgpu ']
}
but it means to select case which has 5D but isn't related to webgpu, this kind of cases are not our target.